### PR TITLE
Add `dev` tasks to packages with unit tests

### PR DESCRIPTION
**Pull request by sndrs**
## What are you changing?

- adds `dev` tasks where meaningful
- adds a `make dev` task that can run from the root

## Why?

- should make it easier to get instant feedback on changes

### CODE REVIEW

**Review by oliverabrahams — APPROVED** [review]
Looks good!

File: libs/@guardian/libs/project.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
 			"outputs": ["{workspaceRoot}/coverage/libs/@guardian/libs"],
 			"options": {
 				"jestConfig": "libs/@guardian/libs/jest.config.ts",
-				"passWithNoTests": true
+				"passWithNoTests": false
```
Comment (jamesmockett) [inline libs/@guardian/libs/project.json:37]: Assume this is intentional as it's `true` for other projects? (Clearly there _are_ tests for libs anyway.)

Comment (sndrs) [reply libs/@guardian/libs/project.json:37]: doh! that slipped through from #1131

Comment (jamesmockett) [reply libs/@guardian/libs/project.json:37]: If #1131 is going in next then there's probably no point changing this back 👍

**Review by oliverabrahams — APPROVED** [review]
LGTM

**Pull request merged**